doc/dev/developer_guide/tests-windows.rst

```diff
@@ -0,0 +1,126 @@
+Jenkins "Status" page.
+
+.. image:: ../../images/windows_ci_status_page.png
+      :align: center
+
+The Windows CI artifacts can be downloaded as a zip archive or viewed inside
+the browser. Click the "artifacts" button to see the contents of the artifacts
+folder.
+
+.. image:: ../../images/windows_ci_artifacts.png
+      :align: center
+
+Artifact contents:
+
+* client/ - Ceph client-side logs (Windows)
+    * eventlog/ - Windows system logs
+    * logs/ - Ceph logs
+    * ceph-windows.conf - Ceph configuration file
+* cluster/ - Ceph server-side logs (Linux)
+    * ceph_logs/
+    * journal
+* test_results/
+    * out/ - raw and xml test output grouped by the test executable
+    * test_results.html - aggregated test report (html)
+    * test_results.txt - aggregated test report (plaintext)
+
+We're using the subunit format and associated tools to aggregate the test results,
+which is especially handy when running a large amount of tests in parallel.
+
+The aggregated test report provides a great overview of the failing tests.
+Go to the end of the file to see the actual errors::
+
+    {0} unittest_mempool.mempool.bufferlist_reassign [0.000000s] ... ok
+    {0} unittest_mempool.mempool.bufferlist_c_str [0.006000s] ... ok
+    {0} unittest_mempool.mempool.btree_map_test [0.000000s] ... ok
+    {0} ceph_test_dokan.DokanTests.test_mount [9.203000s] ... FAILED
+
+    Captured details:
+    ~~~~~~~~~~~~~~~~~
+        b'/home/ubuntu/ceph/src/test/dokan/dokan.cc:136'
+        b'Expected equality of these values:'
+        b'  wait_for_mount(mountpoint)'
+        b'    Which is: -138'
+        b'  0'
+        b''
+        b'/home/ubuntu/ceph/src/test/dokan/dokan.cc:208'
+        b'Expected equality of these values:'
+        b'  ret'
+        b'    Which is: "ceph-dokan: exit status: -22"'
+        b'  ""'
+        b'Failed unmapping: Y:\\'
+    {0} ceph_test_dokan.DokanTests.test_mount_read_only [9.140000s] ... FAILED
+
+The html report is also convenient, grouping the test results by test suite
+(test binary). For security reasons it isn't rendered by default but it can be
+downloaded and viewed:
+
+.. image:: ../../images/windows_ci_html_report.png
+      :align: center
+
+Frequently asked questions
+==========================
+
+1. Why is the Windows CI job the only one that fails on my PR?
+
+Ceph integration tests are normally performed through Teuthology on the Ceph
+Lab infrastructure. These tests are triggered on-demand by the Ceph QA
+team and do not run automatically for every submitted pull request.
+
+Since the Windows CI job focuses only on the client-side Ceph components,
+it can run various integration tests in a timely manner for every pull request
+on GitHub. **In other words, it runs various librados, librbd and libcephfs
+tests that other checks such as "make check" do not.**
+
+For this reason, the Windows CI often catches regressions that are missed by the
```

Comment:
Nit: Might want to mention explicitly that the Windows job builds Ceph in Release rather than Debug mode and with a different set of `-D` options passed to CMake, disabling many of the things/components that are enabled in other CI checks.  This sometimes leads to FTBFS just for the Windows job due to missing ifdefs and such in the code.

Reply:
Good point, I'd also mention the toolchain version as well as other dependencies such as boost.